app/models/goal.py

```diff
@@ -3,3 +3,15 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+
+    def to_dict(self):
+        return {
+            "id": self.id,
+            "title": self.title
+        }
+
+    @classmethod
+    def from_dict(cls, goal_data):
+        new_goal = Goal(title=goal_data["title"])
+        return new_goal
```

Comment:
You can use `cls` as the class to make it more generic.


```suggestion
    def from_dict(cls, goal_data):
        new_goal = cls(title=goal_data["title"])
        return new_goal
```